lithops/serverless/backends/aws_lambda/config.py

```diff
@@ -64,74 +64,76 @@
 
 
 def load_config(config_data):
-    if 'aws' not in config_data:
-        raise Exception("'aws' section is mandatory in the configuration")
+    if "aws" not in config_data:
+        config_data["aws"] = {}
 
-    if not {'access_key_id', 'secret_access_key'}.issubset(set(config_data['aws'])):
-        raise Exception("'access_key_id' and 'secret_access_key' are mandatory under the 'aws' section of the configuration")
+    if "secret_access_key" in config_data["aws"] or "access_key_id" in config_data["aws"]:
+        logger.warning("Using 'secret_access_key' and 'access_key_id' in lithops configuration is not recommended "
+                       "- Use boto3 configuration file in ~/.aws or environment variables instead "
+                       "(https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html)")
 
-    if not config_data['aws_lambda']:
-        raise Exception("'aws_lambda' section is mandatory in the configuration")
+    if not config_data["aws_lambda"]:
+        raise Exception("\"aws_lambda\" section is mandatory in the configuration")
 
-    temp = copy.deepcopy(config_data['aws_lambda'])
-    config_data['aws_lambda'].update(config_data['aws'])
-    config_data['aws_lambda'].update(temp)
+    # temp = copy.deepcopy(config_data["aws_lambda"])
+    config_data["aws_lambda"].update(config_data["aws"])
+    # config_data["aws_lambda"].update(temp)
```

Comment:
The reason of this roundtrip is to make sure that the most specific config is always applied. For example, if you set region both in `aws` and `aws_lamda`, the `aws_lamda` region must be applied. I don't know if there is a better way to make sure the most specific config is applied and not overwritten by the `.update()`, but by commenting those 2 lines, if you have a region set in your `aws` section of the config, for example `us-east1`, and then you do this in your code: `fexec = lithops.FunctionExecutor(region="eu-west2")`, the `us-east1` region will always overwrite the region you explicitly set in the function executor.